spacy/__init__.py

```diff
@@ -14,6 +15,12 @@
 from .errors import Warnings, deprecation_warning
 from . import util
 
+if __version__ >= '2.1.0' and sys.maxunicode <= 65535:
```

Comment:
```suggestion
if sys.maxunicode == 65535:
```

We don't actually need to check the version here, since this code will only be included if the user is running a relevant version anyways. And we might as well do `==`, since there are really only two values. If the result was any other number, it'd be _super_ confusing anyways.